axis_indices: [1,3]
scale: 400.0
motor_count: 2
control_freq: 50.0
